src/Speeds/ScoreSpeeds.qml

```diff
@@ -41,7 +41,9 @@ Rectangle {
                     intervalsListModel.insert(0, {
                                                   "name": m.Name,
                                                   "path": JSON.stringify(m.Path),
-                                                  "speedValue": JSON.stringify(m.Speed) * 720 / 6
+                                                  "speedValue": JSON.stringify(m.Speed) * 720 / 6,
```

Comment:
... isn't it weird to convert a number to string, and then multiply it by a number again ?

Reply:
yes, I changed it !